client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+} from 'state/action-types';
+
+export const status = createReducer( null, {
+	[ EMAIL_VERIFY_REQUEST ]: () => 'requesting',
+	[ EMAIL_VERIFY_REQUEST_SUCCESS ]: () => 'sent',
+	[ EMAIL_VERIFY_REQUEST_FAILURE ]: () => 'error',
+} );
```

Comment:
At this point I like to point out that `createReducer()` is adding code without adding value.

```js
export const status = ( state = null, { type } ) => get( {
	[ EMAIL_VERIFY_REQUEST ]: 'requesting',
	…
}, type, state );
```